definitions/tools/unaligned_seq_fda_stats.cwl

```diff
@@ -251,6 +262,15 @@ requirements:
                     }
                     
                     
+                    # prints out the source file information
+                    printf "\n\n[Input file information]";
+                    foreach my $path (@paths)
+                    {
+                        my ($name, $dir, $ext) = fileparse($path, qr/\.[^.]*/);
+                        printf "\n%s\t%s", $name . $ext, $dir;
```

Comment:
If you don't care about the extension, can simplify this:
```suggestion
                        my ($name, $dir) = fileparse($path);
                        printf "\n%s\t%s", $name, $dir;
```

Reply:
These comments are good points. I am going to clean them up in the source codes as Tom suggests, and also removing comment lines that are not in the right place.